### ISSUE TITLE
Customize tag for Col

### ISSUE THREAD

**Issue by juangl** [reactstrap/reactstrap#314]
Could I render a `<Col>` with an `aside`, `nav` or `main` tag instead a `div`?

**Comment by TheSharpieOne** [comment]
You can use the `tag` prop, such as `<Col tag="aside">`

**Comment by juangl** [comment]
Is it a recently added prop? I'm using `Reactrap 4.0.1` and I get:

```
Warning: Unknown prop `tag` on <div> tag. Remove this prop from the element. For details, see 
```

**Comment by TheSharpieOne** [comment]
Sorry, apparently `Col` is not like the other components, it has [`div` hard-coded](https://github.com/reactstrap/reactstrap/blob/4.1.0/src/Col.js#L93) as the element.
@eddywashere was there a reason for this or should it be updated to support the `tag` prop like the other components?

**Comment by eddywashere** [comment]
Missed it when updating components. +1 to add. Also worth checking any other missing tag props.

**Comment by eddywashere** [comment]
released in 4.1.1
